vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go

```diff
@@ -16090,16 +16090,19 @@ func (x *ISCSIVolumeSource) CodecEncodeSelf(e *codec1978.Encoder) {
 		} else {
 			yysep2 := !z.EncBinary()
 			yy2arr2 := z.EncBasicHandle().StructToArray
-			var yyq2 [7]bool
+			var yyq2 [10]bool
```

Comment:
I thought we did for the bits we expected to generate (proto, deep copy, conversion, etc). Previously, we removed generated ugorji in origin, and so we didn't care about generating it.